tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1289,7 +1289,7 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestWaterCooledReform)
                     1,
                     1.0);
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 4.83);
+    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 4.95);
```

Comment:
IPLV values change due to correction to the condenser entering water temperature per the AHRI STD. This change is higher than the other unit tests IPLV value because of the iterative procedure used to determine evaporator and condenser outlet temperature, which in turn are used in the IPLV calculation for reformulated chiller.